middleware/authz.go

```diff
@@ -0,0 +1,41 @@
+package middleware
+
+// This plugin is based on Casbin: an authorization library that supports ACL, RBAC, ABAC
+// View source at:
+// https://github.com/hsluoyz/casbin
+
+import (
+	"net/http"
+
+	"github.com/hsluoyz/casbin"
```

Comment:
It seems that we are going to make it a separate package. I made one here: https://github.com/casbin/chi-authz

Maybe you want to put it in your ``github.com/pressly/middleware``, but it seems not been created yet.